lightning/backend/tidb.go

```diff
@@ -42,14 +43,15 @@ type tidbEncoder struct {
 
 type tidbBackend struct {
 	db *sql.DB
+	OnDuplicate string
 }
 
 // NewTiDBBackend creates a new TiDB backend using the given database.
 //
 // The backend does not take ownership of `db`. Caller should close `db`
 // manually after the backend expired.
-func NewTiDBBackend(db *sql.DB) Backend {
-	return MakeBackend(&tidbBackend{db: db})
+func NewTiDBBackend(db *sql.DB, OnDuplicate string) Backend {
```

Comment:
```suggestion
func NewTiDBBackend(db *sql.DB, onDuplicate string) Backend {
```

Variable names should never be in uppercase.

Reply:
my apologies, I simply delete prefix of `replaceOnDuplicate` and forget Go's semantic 😭